src/components/schemas/404.yaml

```diff
@@ -0,0 +1,8 @@
+type: object
+properties:
+  message:
+    enum:
+      - Resource not found
+    type: string
+    description: >
+      This error means the requested resource was not found.
```

Comment:
This error means the requested resource was not found or the requester doesn't have sufficient access

Reply:
check this with @snarayanank2 once

Reply:
> This error means the requested resource was not found or the requester doesn't have sufficient access
This seems right to me, because if the role isn't right it throws 401, but if its from some other org or something, its 404 only. right @snarayanank2 ?

Reply:
fixed this